docs/adr/0005-rapid-releases-and-moving-towards-a-monorepo.md

```diff
@@ -0,0 +1,99 @@
+# Rapid releases and moving towards a monorepo
+
+* Status: proposed
+* Deciders: ?
```

Comment:
What do you think about adding a concrete list of deciders, or at least mention "Relman, Releng, Android, iOS, app services"?

Reply:
Sounds great.  I'll start this by putting @rvandermeulen for Relman, and just listing the teams for the others for now.